139_Client_Server/python_server.py

```diff
@@ -0,0 +1,63 @@
+import socket # Gives us #socket.socket() and socket.sock_stream()
+import threading # For multiple connections
+
+clients = []
+
+
+# Thread for each client
+def serve(client_connect, client_address, client_id):
+  global clients
```

Comment:
It's better to avoid global vars, and in this case, since you are not reassigning to it, you don't even need this line.
```suggestion
```